packages/expo-localization/android/src/main/java/expo/modules/localization/LocalizationUtils.kt

```diff
@@ -32,3 +35,17 @@ fun getCurrencyCode(locale: Locale): String? {
     currency?.currencyCode
   }.getOrNull()
 }
+
+fun getRegionCode(locale: Locale): String? {
+  val miuiRegion = getSystemProperty("ro.miui.region")
+  return if (!TextUtils.isEmpty(miuiRegion)) {
```

Comment:
```suggestion
  return if (!miuiRegion.isEmpty()) {
```
I think you can use `isEmpty` function directly.